### PR TITLE
Add a dockerized dev environment setup for Student Net.

### PR DESCRIPTION
**Pull request by aalekhpatel07**
Includes some instructions for setup and usage.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  app:
+    image: aalekhpatel07/student-net:latest
```
Comment (xTrig) [inline docker-compose.yml:4]: Maybe we should leave this as
`build: .`
So we can make changes to the Dockerfile if necessary and don't rely on the shared version.

Comment (aalekhpatel07) [reply docker-compose.yml:4]: Sure! That sounds good.

**Review by xTrig — APPROVED** [review]

**Comment by Meharban-Singh** [comment]
It looks like this is using TypeScript. I am not sure if we would wanna use TypeScript with the project. I mean I love TS but this might be too much for students to learn at the same time while learning new frameworks AND while learning what 'backend' and 'frontend' actually is. What do you think @xTrig 

Does this force us to use TS @aalekhpatel07?

**Comment by aalekhpatel07** [comment]
> It looks like this is using TypeScript. I am not sure if we would wanna use TypeScript with the project. I mean I love TS but this might be too much for students to learn at the same time while learning new frameworks AND while learning what 'backend' and 'frontend' actually is. What do you think @xTrig
> 
> Does this force us to use TS @aalekhpatel07?

Yes, the project was bootstrapped with `--typescript` flag. If it helps, I've changed that to JS now.

**Review by Meharban-Singh — APPROVED** [review]

**Pull request merged**